frontend/src/components/PLAYGROUND/hooks/useRenderTableCell/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { Link } from "@mui/material";
+import { GridRenderEditCellParams } from "@mui/x-data-grid";
+import { useState, ReactNode, useEffect, SetStateAction } from "react";
+import { useNavigate } from "react-router-dom";
+
+export const useRenderTableCell = () => {
+  const [selectedRow, setSelectedRow] = useState<null | { id: number }>(null);
+
+  const navigate = useNavigate();
```

Comment:
just curious why you would equate navigate instead of just using the function?

Reply:
https://reactrouter.com/en/main/hooks/use-navigate

Reply:
That's how it's documented.